lib/src/test/java/space/taran/arklib/user/tags/RootTagStorageTest.kt

```diff
@@ -0,0 +1,52 @@
+package space.taran.arklib.user.tags
+
+import dev.arkbuilders.arklib.data.index.ResourceIndex
+import dev.arkbuilders.arklib.user.tags.Tag
+import dev.arkbuilders.arklib.user.tags.TagStorage
+import dev.arkbuilders.arklib.user.tags.Tags
+import junit.framework.TestCase.assertEquals
+import kotlinx.coroutines.test.runTest
+import org.junit.Before
+import org.junit.Test
+import space.taran.arklib.utils.mockLog
+import space.taran.arklib.utils.TestFiles
+import space.taran.arklib.utils.TestRepo
+import java.util.UUID
+
+class RootTagStorageTest {
+
+    val root = TestFiles.root1
+    lateinit var index: ResourceIndex
+    lateinit var tagStorage: TagStorage
+
+    @Before
+    fun before() = runTest {
+        System.loadLibrary("arklib")
+        mockLog()
+        TestFiles.randomFile(root)
```

Comment:
Where is this random file being deleted?

Reply:
Here a temporary file is also created, but I will add its removal